### PR TITLE
Add FreeBSD instructions

### PR DESCRIPTION
**Pull request by norrland**
Update README with instructions for FreeBSD.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,6 +15,30 @@ docker run -p 6667:6667 42wim/matterircd:latest -bind 0.0.0.0:6667
...
+# make install clean
+```
+
+Enable the service.
+```
+echo "matterircd_enable="YES" >> /etc/rc.conf
+```
+Copy the default configuration and modify to your needs.
+```
+# cp /usr/local/etc/matterircd/matteircd.toml.sample /usr/local/etc/matterircd/matteircd.toml
```
Comment (42wim) [inline README.md]: small typo here twice, `matteircd.toml`  => `matterircd.toml`

**Comment by 42wim** [comment]
Hi, thanks for the PR!
If you could fix the small typo's, I'll merge it, thanks!

**Comment by norrland** [comment]
@42wim, Done!

**Comment by 42wim** [comment]
Thanks!

**Pull request merged**